core-blocks/table/editor.scss

```diff
@@ -1,11 +1,30 @@
 .wp-block-table {
+	table {
+		border-collapse: collapse;
+		width: 100%;
+	}
+
 	td,
 	th {
 		padding: 0.5em;
 		border: 1px solid currentColor;
 	}
 
-	td[data-mce-selected="1"], th[data-mce-selected="1"] {
-		background-color: $light-gray-500;
+	td[data-mce-selected="1"],
+	th[data-mce-selected="1"] {
+		background-color: $light-gray-300;
 	}
 }
+
+// Style the resize handles
+.ephox-snooker-resizer-rows {
```

Comment:
Should these styles be applied only within the context of the table block? As implemented, these will impact any table on the screen, regardless of within the block. Even if they're desirable globally, I don't think the styles should be implemented here (since a block's styles shouldn't be impacting the page as a whole).

Reply:
I wish I could scope these to be table block specific only. However these elements are inserted into the root of the DOM. 

I can move them to the main stylesheet if you like?